bottlecap/src/main.rs

```diff
@@ -156,6 +159,8 @@ fn main() -> Result<()> {
                     deadline_ms,
                     invoked_function_arn
                 );
+                logs_agent.set_function_arn(invoked_function_arn);
```

Comment:
it could be removed if you can build the arn from env (see previous comment)
and arn would not need to be an Option the LogAgent